### PR TITLE
bump version with date (for xcp 57600 baud rate access grant)

### PR DESCRIPTION
**Pull request by hphu3**
^^^
needed for https://github.com/inventables/easel/pull/7423

### CODE REVIEW

**Review by rodovich — COMMENTED** [review]
For the sake of testing at 57600 baud, I think we also would need to build with a different value of

https://github.com/inventables/grbl-xcp/blob/564ecb2638e652a15de49f982a186d9cf1c44aef/grbl/config.h#L42

**Comment by hphu3** [comment]
> For the sake of testing at 57600 baud, I think we also would need to build with a different value of
> 
> https://github.com/inventables/grbl-xcp/blob/564ecb2638e652a15de49f982a186d9cf1c44aef/grbl/config.h#L42

I thought easel was responsible for the baud rate number (Since it has the controller cycling between different rates)

File: grbl/grbl.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #define grbl_h
 
 // Grbl versioning system
-#define GRBL_VERSION "1.1h-XCP"
+#define GRBL_VERSION "1.1h-XCP-20220314"
```
Comment (neilpa-inv) [inline grbl/grbl.h:25]: The next `#define` already sets the date. If we're globally including it now then it should be removed. However, lets keep the same format as was previously used [here](https://github.com/inventables/grbl-xcp/blob/564ecb2638e652a15de49f982a186d9cf1c44aef/grbl/report.c#L371) (which also needs to be tweaked).

Comment (neilpa-inv) [reply grbl/grbl.h:25]: Can you also update the [comment here](https://github.com/inventables/grbl-xcp/blob/master/bootloader/combine.sh#L22)

**Review by neilpa-inv — CHANGES_REQUESTED** [review]

**Review by neilpa-inv — APPROVED** [review]

**Pull request merged**